app/src/main/res/values/strings.xml

```diff
@@ -120,4 +120,5 @@
         make an image</string>
     <string name="action_agree">AGREE</string>
     <string name="action_dis_agree">DIS AGREE</string>
+    <string name="action_add_photo">Add Photo</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.action_add_photo` appears to be unused
 Unused resources make applications larger and slow down builds.